openfisca_uk/parameters/hmrc/benefit/child_benefit/amount/additional.yaml

```diff
@@ -0,0 +1,45 @@
+description: Child Benefit amount for each additional child
```

Comment:
Should this path be `hmrc/benefit/child_benefit` or `hmrc/child_benefit`?

In openfisca-us I think we're moving to these folders being either agencies or programs. Child Benefit is listed as a top level category in the [HMRC section of gov.uk](https://www.gov.uk/government/organisations/hm-revenue-customs), so I think we could skip `/benefit`. Notably, Tax Credits are also their own section, so I think that layer (rather than WTC and CTC being under HMRC) still makes sense.

Reply:
Moved to hmrc/child_benefit, thanks